docs/content/reference/exploiters/powershell.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "PowerShell"
+date: 2021-08-24T12:19:21+03:00
+draft: false
+tags: ["exploit", "windows"]
+---
+
+### Description
+
+PowerShell Remoting is a brute-force exploit that uses PowerShell Remoting Protocol (PSRP) and
+Windows Remote Management(WinRM) services to propagate to a victim.
+
+### PowerShell Remoting Protocol
+
+PowerShell Remoting Protocol uses Windows Remote Management service, which is Microsoft implementation of Web Services
+for Management (WS-Management) protocol, to allow users to run PowerShell commands on remote computers.
+
+PowerShell Remoting and WinRM listens on the following ports:
+1. HTTP: 5895
+2. HTTPS: 5896
+
+By default, PowerShell Remoting only allows connections from members of the Administrators group. Sessions are launched
+under the user's context, so all operating system access controls applied to individual users and groups continue to apply
+to them while connected over PowerShell Remoting.
+
+On private networks, the default Windows Firewall rule for PowerShell Remoting accepts all connections. On public networks,
+the default Windows Firewall rule allows PowerShell Remoting connections only from within the same subnet. You have to
+explicitly change that rule to open PowerShell Remoting to all connections on a public network.
+
+More on [PowerShell Remoting Protocol]("https://docs.microsoft.com/en-us/powershell/scripting/learn/remoting/winrmsecurity?view=powershell-7.1").
+
+### Windows Remote Management
+
+Windows Remote Management (WinRM) is the Microsoft implementation of WS-Management Protocol, a standard Simple Object
+Access Protocol (SOAP)-based, firewall-friendly protocol that allows hardware and operating systems, from different
+vendors, to interoperate.
+
+More on [Windows Remote Management]("https://docs.microsoft.com/en-us/windows/win32/winrm/portal").
```

Comment:
This is just details about PSRP and WinRM from the documentation. If someone wants to know about them in detail, they can look up the documentation themselves. We don't need to replicate the information in our documentation.